crates/hc/src/bin/hc.rs

```diff
@@ -3,9 +3,12 @@ use holochain_cli as hc;
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
-    if std::env::var_os("RUST_LOG").is_some() {
-        holochain_trace::init_fmt(holochain_trace::Output::Log).ok();
+    // This is necessary to ensure deprecation warnings in holochain_cli_bundle are displayed by default
+    if std::env::var_os("RUST_LOG").is_none() {
+        std::env::set_var("RUST_LOG", "holochain_cli_bundle=warn");
```

Comment:
Not sure if this is the right approach, or if I should simply `println!` the warning in hc bundle

Reply:
I like the idea of a default log config, but probably not here no. We've talked previously about having a default log config in `holochain_trace`. Something like `warn,wasmer_compiler_cranelift=off`. Then if you don't want the default logs you either have to pick something or set `RUST_LOG=off`. I'd be pleased if we did something like that in `holochain_trace`. Otherwise it would also be fine to `eprintln` where we have deprecation messages to make sure people see them